### PR TITLE
Fix unit tests AB#9965

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#9965](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9965)

* [ ] Enhancement
* [x] Bug
* [ ] Documentation

## Description

Fixes and reenable front end unit tests.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [x] Unit Tests Updated
* [ ] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES | NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/constants/resulttype.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const enum ResultType {
+export enum ResultType {
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/constants/resulttype.ts:1]: @FuriousLlama @tiagograf-AOT @sslaws  should we use const or without const?

**Review by leobcgov — CHANGES_REQUESTED** [review]
waiting for team's confirmation

**Review by sslaws — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/constants/resulttype.ts (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/WebClient/src/ClientApp/src/constants/resulttype.ts:1]: it breaks the test

Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/constants/resulttype.ts:1]: well that is a sad change... Was that the line the broke it?

**Review by FuriousLlama — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/constants/resulttype.ts (hunk 1 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/constants/resulttype.ts:1]: approved then :-)

**Review by leobcgov — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/constants/resulttype.ts (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/WebClient/src/ClientApp/src/constants/resulttype.ts:1]: yes apparently, it also seems that theres a cache in play but i don't understand is how a cache is possibly being applied to jest.

**Pull request merged**